tools/gen_addon_readme.template

```diff
@@ -0,0 +1,86 @@
+{%- macro fragment(name, title, sub='=') %}
+{%- if name in fragments %}
+{{- title }}
+{{ sub * title|length }}
+
+{{ fragments[name] }}
+{% endif %}
+{%- endmacro -%}
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+{% for (image, target, alt) in badges -%}
+.. |badge{{ loop.index }}| image:: {{ image }}
+    :target: {{ target or '#' }}
+    {%- if alt %}
+    :alt: {{ alt }}
+    {%- endif %}
+{% endfor %}
+{% for _ in badges %}|badge{{ loop.index }}| {% endfor %}
+
+{{ '=' * manifest.name|length }}
+{{ manifest.name }}
+{{ '=' * manifest.name|length }}
+
+{{ fragments.get('DESCRIPTION', '') }}
+.. contents::
+   :local:
+
+{{ fragment('INSTALL', 'Installation') }}
+{{- fragment('CONFIGURE', 'Configuration') }}
+{{- fragment('USAGE', 'Usage') }}
+{{- fragment('DEVELOP', 'Development') }}
+{{- fragment('ROADMAP', 'Known issues / Roadmap') -}}
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/{{ repo_name }}/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+{% if authors -%}
+Authors
+-------
+
+{% for author in authors -%}
+* {{ author }}
+{% endfor %}
+{% endif -%}
+
+{{ fragment('CONTRIBUTORS', 'Contributors', sub='-') }}
+{{- fragment('CREDITS', 'Other credits', sub='-') -}}
+Maintainers
+-----------
+
+This module is maintained by the OCA.
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+{%- if manifest.maintainers %}
+{% for maintainer in manifest.maintainers %}
+.. |maintainer-{{ maintainer }}| image:: https://github.com/{{ maintainer }}.png?size=50px
+    :target: https://github.com/{{ maintainer }}
+    :alt: {{ maintainer}}
+{%- endfor %}
+
+Current maintainer{% if manifest.maintainers|length > 1 %}s{% endif %}: 
+
+{% for maintainer in manifest.maintainers %}|maintainer-{{ maintainer }}| {% endfor -%}
+{% endif %}
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit https://odoo-community.org.
```

Comment:
IMO this link is quite vague for the goal proposed.
Perhaps say "You are welcome to contribute. To learn how please visit https://odoo-community.org/page/Contribute".

Reply:
Good idea. I was also thinking to add a link to the repo here.